main.sh

```diff
@@ -235,16 +241,16 @@ do_strip() {
 case "${INPUT_TARGET:-}" in
     '')
         build
-        target_dir="${target_dir}/release"
+        target_dir="${target_dir}/${INPUT_PROFILE}"
```

Comment:
Some profiles use different directories than the profile name, so you have to handle those.

- release, bench -> release
- dev, test -> debug
- (other) -> (profile name)

See also cargo-llvm-cov code linked in the above comment.

Reply:
I updated the code so it'll rename the profile for the special cases